### ISSUE TITLE
Unable to create another project

### ISSUE THREAD

**Issue by abdalem** [clearflask/clearflask#123]
Hi !

First, thanks about this awesome tool. I have an issue about creating a second project.

![image](https://github.com/clearflask/clearflask/assets/19971392/07d5b62c-e256-4a81-8c92-631ccff7cd4f)

As you can see in the above image, it seems that the domain is conflicting with my other projet. But the portal domain input is not editable. So I have change my other project domain and subdomain as follow:

![image](https://github.com/clearflask/clearflask/assets/19971392/ce89cfad-a3b1-4b49-8af4-d65149b0f3dc)

But I am still getting the issue and nothing seems to fix it.

**Comment by matusfaro** [comment]
Ah yes, when you change the subdomain or a custom domain, we keep the old one pointing to your project for a day just in case you want to switch it back. After the one day, you are free to use it for another project.

If you want to do this immediately, you need to change the configuration property to reduce this time using:
```
com.smotana.clearflask.store.impl.DynamoProjectStore$Config.slugExpireAfterMigration=PT1S
```
Then restart your server or just wait for the config to be reloaded automatically within 1 minute

**Comment by abdalem** [comment]
@matusfaro Great thanks !

Do you know why the input `portal domain` act differently on self host and on cloud saas ?
![image](https://github.com/clearflask/clearflask/assets/19971392/a29ffcb2-9f14-4b8f-be49-37da3d39facd)

This screenshot is from Saas, which it is more clearly understandable and also editable unlike the self hosted one which is clearly not editable and not displaying same thing. Maybe a config ?

**Comment by abdalem** [comment]
@matusfaro Thank you for your feedback !

```
com.smotana.clearflask.store.impl.DynamoProjectStore$Config.slugExpireAfterMigration=PT1S
``` 

By the way I've added this into config but it does not seems to work. I have setup abeecrm.feedback.abbi.tech, in subdomain & domain, but:
- feedback.abbi.tech is still pointing to the project
- adding a project now return a 500 error

**Comment by matusfaro** [comment]
This is now fixed in [1.6.9](https://github.com/clearflask/clearflask/releases/tag/1.6.9)
